requirements/lint.txt

```diff
@@ -8,3 +8,6 @@
 # are pinned to prevent unexpected linting failures when tools update)
 ruff==0.6.7
 mypy==1.11.2
+
+# Required for type stubs
+freezegun==1.5.1
```

Comment:
By the way, for a similar reason (and especially that types can effectively be different across Python versions), I've switched typing checks in my private projects from "linting" environment to the main test environment, and therefore I run them for every impl separately.